pypeit/find_objects.py

```diff
@@ -1011,26 +1010,103 @@ def find_objects_pypeline(self, image, ivar, std_trace=None,
         nperorder =  self.par['reduce']['findobj']['maxnumber_std'] if self.std_redux \
             else self.par['reduce']['findobj']['maxnumber_sci']
 
-        sobjs_ech = findobj_skymask.ech_objfind(
-            image, ivar, self.slitmask, self.slits_left, self.slits_right,
-            self.order_vec, self.reduce_bpm, det=self.det,
-            spec_min_max=np.vstack((self.slits.specmin, self.slits.specmax)),
-            inmask=inmask, ncoeff=self.par['reduce']['findobj']['trace_npoly'],
-            hand_extract_dict=manual_extract_dict, plate_scale=plate_scale,
-            std_trace=std_trace,
-            specobj_dict=specobj_dict,
-            snr_thresh=self.par['reduce']['findobj']['snr_thresh'],
-            show_peaks=show_peaks, show_fits=show_fits,
-            trim_edg=self.par['reduce']['findobj']['find_trim_edge'],
-            fwhm=self.par['reduce']['findobj']['find_fwhm'],
-            use_user_fwhm=self.par['reduce']['extraction']['use_user_fwhm'],
-            nperorder=nperorder,
-            maxdev=self.par['reduce']['findobj']['find_maxdev'],
-            max_snr=self.par['reduce']['findobj']['ech_find_max_snr'],
-            min_snr=self.par['reduce']['findobj']['ech_find_min_snr'],
-            nabove_min_snr=self.par['reduce']['findobj']['ech_find_nabove_min_snr'],
-            box_radius=self.par['reduce']['extraction']['boxcar_radius'],  # arcsec
-            show_trace=show_trace, objfindQA_filename=objfindQA_filename)
+        orig = True
```

Comment:
Are you planning to delete the orig=False control flow below?

Reply:
yes